.github/workflows/versionincrement.yaml

```diff
@@ -0,0 +1,87 @@
+---
+name: Version Increment
+
+on:
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: Log level
+        required: true
+        default: warning
+        type: choice
+        options:
+          - info
+          - warning
+          - debug
+
+jobs:
+  plugin-version-increment-sync:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        entry:
+          - {repo: alerting}
```

Comment:
Nit: auto-format this file with your editor, there should be a space after `{`, so `- { repo: alerting }`.

Reply:
I just see us have spaces everywhere else in other yml files.

Reply:
I dont think so @dblock https://github.com/opensearch-project/opensearch-build/pull/2291/commits/b1f75ce0d049cf18ca678f1f779f9169bc321a36 `yamllint` fails with `24:14 [braces] too many spaces inside braces`